src/main/java/com/eclipsesource/v8/utils/MemoryManager.java

```diff
@@ -92,7 +92,7 @@ public boolean isReleased() {
      */
     public void release() {
         v8.getLocker().checkThread();
-        if (released) {
+        if (isReleased()) {
```

Comment:
No need to change this as part of this commit. We can use the field without the accessor for internal calls.